nilearn/reporting/html_doc.py

```diff
@@ -0,0 +1,153 @@
+import os
```

Comment:
I would prefer the full name `html_document`; when I read "doc" I understand "documentation"